lib/executor/helpers.go

```diff
@@ -96,6 +151,10 @@ func getIterationRunner(
 			return false
 		default:
 			if err != nil {
+				if handleInterupt(ctx, err) {
+					executionState.AddInterruptedIterations(1)
+					return false
+				}
```

Comment:
Looking at the code nothing uses the return value. This seems to not matter as the contexts are getting checked constantly, but it is probably a good idea to figure out if we need to use it somehow

cc @imiric @na--

Reply:
Yeah, we might make a small PR to refactor this, but it might be needed eventually (e.g. in https://github.com/k6io/k6/issues/1250 or https://github.com/k6io/k6/issues/877) and is so small that it's probably fine as it is.